core/src/pretty.rs

```diff
@@ -5,6 +5,9 @@ use crate::parser::lexer::KEYWORDS;
 use crate::term::{
     pattern::*,
     record::{Field, FieldMetadata, RecordData},
+    // Because we use `Term::*`, we need to differentiate `Contract` from `Term::Contract`, so we
+    // alias the latter
+    CustomContract as ContractNode,
```

Comment:
But it's called `CustomContract`, not `Contract` :thinking:

Reply:
Ah, but I initially called it `Contract`, and renaming didn't catch that, of course. I'm already working on a follow-up, so will fix it there instead.